packages/komencikit/src/verifyWallet.ts

```diff
@@ -1,8 +1,8 @@
 import { Address, normalizeAddress } from '@celo/base'
 import { Err, Ok, Result } from '@celo/base/lib/result'
 import { ContractKit } from '@celo/contractkit'
+import { ABI as Proxy } from '@celo/contractkit/lib/generated/Proxy'
```

Comment:
This isn't right -- we either extract the bytecode in a constat or we need it from protocol. The ABI in contractkit doesn't contain the bytecode.